client/src/reducers/lessonReducer.ts

```diff
@@ -7,6 +7,9 @@ export interface WordInfo {
     word: string;
     part_of_speech?: string;
     forms?: WordForm[];
+    synonyms?: string[];
+    antonyms?: string[];
+    definition?: string;
```

Comment:
It's fully typed out for the set of implemented features... figured I'd expand it when we get context sentences. Is that what you meant?